sia20:
    # Publish all joint state ---------------------------
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 100

    # Position Controllers ------------------------------
    sia20_arm_controller:
        type: effort_controllers/JointTrajectoryController
        joints:
            - joint_s
            - joint_l
            - joint_e
            - joint_u
            - joint_r
            - joint_b
            - joint_t

        gains:
            joint_s: {p: 100, i: 0.01, d: 10.0}
            joint_l: {p: 100, i: 0.01, d: 10.0}
            joint_e: {p: 100, i: 0.01, d: 10.0}
            joint_u: {p: 100, i: 0.01, d: 10.0}
            joint_r: {p: 100, i: 0.01, d: 10.0}
            joint_b: {p: 100, i: 0.01, d: 10.0}
            joint_t: {p: 100, i: 0.01, d: 10.0}

